opensrp-pnc/src/main/java/org/smartregister/pnc/config/PncConfiguration.java

```diff
@@ -39,6 +46,15 @@ private void setDefaults() {
         if (!builder.pncFormProcessingClasses.containsKey(PncConstants.EventTypeConstants.PNC_VISIT)) {
             builder.pncFormProcessingClasses.put(PncConstants.EventTypeConstants.PNC_VISIT, PncVisitFormProcessing.class);
         }
+
+        if (!builder.pncFormProcessingClasses.containsKey(PncConstants.EventTypeConstants.PNC_CLOSE)) {
+            builder.pncFormProcessingClasses.put(PncConstants.EventTypeConstants.PNC_CLOSE, PncCloseFormProcessing.class);
+        }
+    }
+
+    @NonNull
+    public <T extends VisitScheduler> T getPncVisitScheduler() {
```

Comment:
should be `Class<? extends VisitScheduler>`

Reply:
done